src/phasmophobia-data/data.js

```diff
@@ -1,5 +1,83 @@
+		"Preferes shattering lights over other ghost events",
+		"Very fast when chasing a player during a hunt (2x normal speed)",
+		"Very slow when not chasing anyone during a hunt (1/2 of normal speed)",
+		"Less chance of ghost activity and ghost events when more than 1 player is nearby",
+		"Lower chance of initiating a hunt when more than 1 players are in the same room",
+		"Can initiate a hunt at an avarage sanity of 70%",
+		"Successful answers from the ouja board lowers sanity by 40% instead of 50%",
+		"Ghost event drops sanity by 0.4% per second instead of 0.2%",
+		"This also applies when a player is within 10m during a hunt",
+		"Smudging will make it to no leave the room for 1.5 min",
+		"Can initiate a hunt at an avarage sanity of 80% if players are talking nearby",
+		"During a hunt it only can hear sounds in a radius of 2m",
+		"Walking speed is affected by room temperature when hunting and not chasing",
+		"Faster (1.8 m/s) below 10°C/50°F - slower (~1.4 m/s) above 10°C/50°F",
+		"Shows cold breath during a hunt in rooms with freezing temperature",
+		"D.O.T.S. only visible through a video cam and no players in the room",
+		"Less likely to roam when not hunting",
+		"Always gives the D.O.T.S. evidence on nightmare difficulty",
+		"More sounds on the parabolic mic",
+		"During a hunt footstep sounds can only be heard when 10m or closer (similar to electronic interference)",
+		"Can initiate a hunt when any flames are extinguished regardless of average sanity",
+		"Chance for the ability increases everytime a player is killed",
+		"Lit candles act as crucifixes. Blown out when successfuly prevented a hunt",
+		"Main twin primary stays in the ghost room, makes freezing and spirit box",
+		"Secondary twin wanders around interacting with the environment",
+		"Secondary twin cannot interact with motion sensors, spirit box or make freezing",
+		"Either Twin can initiate a Hunt but not simultaneously",
+		"Crucifix only checks for main twin regardless of which will hunt",
+		"During hunts the main twin is slower (by 10%) and the decoy is faster (by 10%)",
+		"While hunting it has an increased movement speed when it touches electronic equipment",
+		"It doesn't matter if the equipment lies on the floor or held by a player",
+		"Abilities only affected by player equipment and not location electrics like TVs or lights",
+		"Head mounted cameras, motion sensors and sound sensors do not count towards the speed boost",
+		"Can initiate a hunt at an avarage sanity of 70% if there is electrical equipment nearby",
+		"Reduced chance to leave fingerprints",
+		"Low chance to leave fingerprints with 6 fingers",
+		"Fingerprints have a 50% chance to disappear earlier (60s) than of other ghosts (120s)",
+		"Always gives the Fingerprints evidence on nightmare difficulty",
+		"Has ghost orbs as 4th evidence",
+		"Picks a ghost to mimic and changes it occasionally",
+		"Copies all abilities and behaviour of the picked ghost"
+	],
```

Comment:
You dont require those anymore. In order to reduce redundancy, I removed these two arrays from the data.js file in a recent commit. They are now automatically generate at website startup from the ghosts primary and secondary evidences instead.

Reply:
Wow good to know!